cradle-cassandra/src/main/java/com/exactpro/cradle/cassandra/dao/testevents/TestEvenInserter.java

```diff
@@ -58,17 +59,23 @@ public CompletableFuture<AsyncResultSet> insert(TestEventEntity testEvent, Funct
                 .setInt(FIELD_EVENT_COUNT, testEvent.getEventCount())
                 .setLocalDate(FIELD_END_DATE, testEvent.getEndDate())
                 .setLocalTime(FIELD_END_TIME, testEvent.getEndTime())
-                .setBoolean(FIELD_COMPRESSED, testEvent.isCompressed())
-                .setByteBuffer(FIELD_MESSAGES, testEvent.getMessages())
-                .setSet(FIELD_LABELS, testEvent.getLabels(), String.class)
-                .setByteBuffer(FIELD_CONTENT, testEvent.getContent())
+                .setBoolean(FIELD_COMPRESSED, testEvent.isCompressed());
+
+        if (testEvent.getMessages() != null) {
+            builder = builder.setByteBuffer(FIELD_MESSAGES, testEvent.getMessages());
+        }
+
+        if (testEvent.getLabels() != null) {
+            builder = builder.setSet(FIELD_LABELS, testEvent.getLabels(), String.class);
+        }
```

Comment:
question: Does it help us to avoid `null` values in the table? If we do not set the value for a column it will have a null value, I believe. Probably, we need to set some kind of a  default value in this case

Reply:
I expect it will have UNSET value.

Reply:
Okay, got it. Have you already checked that? If it is true then we should leave a comment about why we conditionally set those fields

Reply:
If we set null to value for a column it will have null value and tombstone for the row will be created. If we don't set the value for the column, it also have null value, but tombstone is not created.

Reply:
Could you please share how do check it. And could we add the check into our integration tests (I mean all current tests)